src/builtin_string.cpp

```diff
@@ -1125,6 +1132,27 @@ static int string_split0(parser_t &parser, io_streams_t &streams, int argc, wcha
     return string_split_maybe0(parser, streams, argc, argv, true /* is_split0 */);
 }
 
+static int string_collect(parser_t &parser, io_streams_t &streams, int argc, wchar_t **argv) {
+    options_t opts;
+    opts.chomp_valid = true;
+    int optind;
+    int retval = parse_opts(&opts, &optind, 0, argc, argv, parser, streams);
+    if (retval != STATUS_CMD_OK) return retval;
+
+    auto &buff = streams.out.buffer();
+    arg_iterator_t aiter(argv, optind, streams, false);
```

Comment:
This "false" here is probably the single most important part - that's the thing that disables splitting. It deserves a `/* don't split */` comment.

Reply:
Done